### PR TITLE
Create AudioBooks.com.xml

### PR DESCRIPTION
**Pull request by Giltyhub2**

### CODE REVIEW

File: src/chrome/content/rules/AudioBooks.com.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<ruleset name="AudioBooks.com">
+	<target host="audiobooks.com" />
+	<target host="www.audiobooks.com" />
+	<target host="affiliates.audiobooks.com" />
+	<target host="api.audiobooks.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/AudioBooks.com.xml]: au.audiobooks.com

File: src/chrome/content/rules/AudioBooks.com.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	<target host="blog.audiobooks.com" />
+	<target host="ca.audiobooks.com" />
+	<target host="m.audiobooks.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/AudioBooks.com.xml]: uk.audiobooks.com

File: src/chrome/content/rules/AudioBooks.com.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<ruleset name="AudioBooks.com">
+	<target host="audiobooks.com" />
+	<target host="www.audiobooks.com" />
+	<target host="affiliates.audiobooks.com" />
+	<target host="api.audiobooks.com" />
+	<target host="blog.audiobooks.com" />
+	<target host="ca.audiobooks.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/AudioBooks.com.xml]: images.audiobooks.com

https://images.audiobooks.com/cds/home-press-logos/forbes-white.png

Comment (Bisaloo) [inline src/chrome/content/rules/AudioBooks.com.xml]: covers.audiobooks.com

https://covers.audiobooks.com/images/covers/full/9781529010756.jpg

**Comment by Giltyhub2** [comment]
thanks a ton @Bisaloo, i implemented those changes

**Pull request merged**